scripts/globals/caskets.lua

```diff
@@ -243,8 +98,10 @@ local function timeElapsedCheck(npc)
     local spawnTime   = os.time() + 360000 -- defualt time in case no var set.
     local timeTable   = {0, 0, 0}          -- HOURS,MINUTES,SECONDS.
 
-    spawnTime = npc:getLocalVar("[caskets]SPAWNTIME")
-
+    if npc ~= nil then
+        spawnTime = npc:getLocalVar("[caskets]SPAWNTIME")
+    end
+    
```

Comment:
Thoughts on doing:
```lua
if npc == nil then
    return false
end
```
To immediately abort the rest of the function?

Reply:
thats a good call i'll do that

Reply:
im gonna add a check the see if the casket has a local var set for its spawn time too.

Reply:
Added the the suggested change plus a check for the local var.